src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -259,7 +265,7 @@ class ReferralService(
 
     referralDetailsRepository.saveAndFlush(newDetails)
 
-    if (existingDetails !== null && existingDetails !== newDetails) {
+    if (existingDetails !== newDetails) {
```

Comment:
_verifying understanding_
`existingDetails` can never be null because we are in a "sent referral" context
Is that correct?

Reply:
You that's right @sldblog